packages/uni_app/lib/view/login/login.dart

```diff
@@ -364,46 +335,4 @@ class LoginPageViewState extends State<LoginPageView>
       },
     );
   }
-
-  void _updatePasswordDialog() {
```

Comment:
Still have to see the changes better, but on a first sight, I do not think that this is supposed to be erased.

Reply:
I've just removed to make the linter test stop [complaining](https://github.com/NIAEFEUP/uni/actions/runs/11712953818/job/32624747784), since this function is not used anywhere now. But I can revert the commit or leave it commended if necessary.

Reply:
Look at my upper comment for this